src/Arcus.Messaging.Pumps.ServiceBus/Configuration/IAzureServiceBusTopicMessagePumpOptions.cs

```diff
@@ -21,8 +21,15 @@ public interface IAzureServiceBusTopicMessagePumpOptions
         /// Gets or sets the maximum concurrent calls to process messages.
         /// </summary>
         /// <exception cref="ArgumentException">Thrown when the <paramref name="value"/> is less than or equal to zero.</exception>
-        int? MaxConcurrentCalls { get; set; }
-        
+        int MaxConcurrentCalls { get; set; }
+
+        /// <summary>
+        /// Gets or sets the number of messages that will be eagerly requested from
+        /// Queues or Subscriptions and queued locally, intended to help maximize throughput
+        /// by allowing the processor to receive from a local cache rather than waiting on a service request.
+        /// </summary>
+        public int PrefetchCount { get; set; }
```

Comment:
```suggestion
        int PrefetchCount { get; set; }
```

Reply:
Since we're working on a new major version nonetheless, we can create a breaking change here in the interface. 👍